models/sipnet/R/read_restart.SIPNET.R

```diff
@@ -38,24 +43,18 @@ read_restart.SIPNET <- function(outdir, runid, stop.time, settings, var.names, p
   # last <- which(as.Date(real_time)==as.Date(stop.time))[1]
   last <- which(as.Date(real_time)==as.Date(stop.time))[length(which(as.Date(real_time)==as.Date(stop.time)))]
   
-  params$restart <-c() ## This will be filled with some restart coefficient if above ground wood is in the state variables.
-  
   #### PEcAn Standard Outputs
   if ("AbvGrndWood" %in% var.names) {
     forecast[[length(forecast) + 1]] <- PEcAn.utils::ud_convert(ens$AbvGrndWood[last],  "kg/m^2", "Mg/ha")
     names(forecast[[length(forecast)]]) <- c("AbvGrndWood")
-    
+  }else{
```

Comment:
If AGB is in the var.names, then the root carbon will not be updated into params object.